### PR TITLE
[FIZZ-74] Basic users mapping

### PR DESCRIPTION
**Pull request by lalrg**

### CODE REVIEW

File: src/canvas-extractor/canvas_extractor/mapping/users.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pandas as pd
+
+from . import constants
+
+
+def map_to_udm(users_df: pd.DataFrame) -> pd.DataFrame:
```
Comment (bradbanister) [inline src/canvas-extractor/canvas_extractor/mapping/users.py:11]: Not a strong opinion, but you could rename to something like "map_to_udm_users" and then not have to rename when importing multiple of them.

**Review by bradbanister — COMMENTED** [review]
Looking good so far

File: src/canvas-extractor/canvas_extractor/mapping/users.py (hunk 1 above)
Comment (lalrg) [reply src/canvas-extractor/canvas_extractor/mapping/users.py]: ✔️

File: src/canvas-extractor/canvas_extractor/csv_generation/write.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (bradbanister) [inline src/canvas-extractor/canvas_extractor/csv_generation/write.py:1]: If this is identical to the GC version, maybe put a big comment to that effect so we know to extract to a shared library.  See api/resource_sync.py for an example.

**Review by bradbanister — APPROVED** [review]
Looks good, consider adding code comment if it applies.

File: src/canvas-extractor/canvas_extractor/csv_generation/write.py (hunk 2 above)
Comment (lalrg) [reply src/canvas-extractor/canvas_extractor/csv_generation/write.py:7]: ✔️

**Pull request merged**